cluster-provision/gocli/cmd/run.go

```diff
@@ -56,6 +56,7 @@ func NewRunCommand() *cobra.Command {
 	run.Flags().UintP("nodes", "n", 1, "number of cluster nodes to start")
 	run.Flags().StringP("memory", "m", "3096M", "amount of ram per node")
 	run.Flags().UintP("cpu", "c", 2, "number of cpu cores per node")
+	run.Flags().UintP("secondary-nics", "s", 0, "number of secondary nics to add")
```

Comment:
i'd say we don't need short arg name for this one, s/"s"/""/

Reply:
Yep, was not sure about it, let's just remove.